pr_agent/algo/__init__.py

```diff
@@ -8,6 +8,7 @@
     'gpt-4': 8000,
     'gpt-4-0613': 8000,
     'gpt-4-32k': 32000,
+    'gpt-4-1106-preview': 128000, # 128K, but may be limited by config.max_model_tokens
```

Comment:
**Suggestion:** Since the 'gpt-4-1106-preview' model's token limit might be restricted by the configuration, consider removing the entry from the MAX_TOKENS dictionary and relying solely on the dynamic calculation in 'get_max_tokens' to avoid redundancy and potential inconsistency.
```suggestion
    # 'gpt-4-1106-preview': 128000, # This line is commented out as the token limit is dynamically calculated in get_max_tokens
```